youtube_dl/extractor/common.py

```diff
@@ -1539,6 +1541,59 @@ def extract_multisegment_info(element, ms_parent_info):
         self._sort_formats(formats)
         return formats
 
+    def _extract_html5_media_entries(self, url, webpage):
```

Comment:
This should be called `parse` in order to match the naming convention of the other parse/extract methods since it does not download the webpage but just parses it.